.travis.yml

```diff
@@ -23,19 +18,22 @@ install:
     - pip -q install -U pip
     - pip -q install -r oq-engine/requirements-py36-linux64.txt
     - pip -q install -e oq-engine
-    - docker pull openquake/qgis-builder:3
-    - docker build --build-arg uid=$(id -u) -t openquake/qgis-builder:3 -f docker/Dockerfile docker/
 
 before_script:
     - oq restore oqdata-${BRANCH}.zip ~/oqdata
     - oq webui start 0.0.0.0:8800 --skip-browser &> webui.log &
 
 script:
-    - docker run -ti -e OQ_ENGINE_HOST='http://172.17.0.1:8800' --rm -v $TRAVIS_BUILD_DIR:/io openquake/qgis-builder:3 test
-
-after_success:
-    - docker run -ti --rm -v $TRAVIS_BUILD_DIR:/io openquake/qgis-builder:3 build_apidoc
-    - docker run -ti --rm -v $TRAVIS_BUILD_DIR:/io openquake/qgis-builder:3 build_manual_html
+    - cd scripts
+    - ./run_unit_tests.sh
```

Comment:
Please add a `FIXME` regarding the possibility to use a multi-stage job

Reply:
Done